docs/products/grafana/howto/grafana-authentication-plugins.rst

```diff
@@ -0,0 +1,106 @@
+Grafana authentication plugins
+##############################
+
+Configure Google, GitHub or GitLab authentication integration to Aiven Grafana
+-------------------------------
+
+Grafana supports multiple different authentication plugins, in addition to built-in username and password authentication.
+
+Aiven Grafana has support for Google, GitHub and GitLab authentication. Configuring authentication integration is currently available in Aiven console, or can be configured either with the API or with `Aiven command-line client <https://github.com/aiven/aiven-client>`_ (``avn``).
+
+Please note that enabling this feature requires applying a maintenance upgrade for Grafana services created before September 2018.
+
+Google Authentication
+---------------------
+
+First, create your Google OAuth keys by following `this article <http://docs.grafana.org/auth/google/>`_ and make a note of your client ID and client secret. Then open your Aiven Grafana service and navigate to `Advanced Configuration <https://help.aiven.io/en/articles/3601906-advanced-configuration-in-aiven-console>`_ at the bottom of the page. Configure the following options in the advanced configuration:
+
+* ``auth_google.allowed_domains``: set to your G-suite domain
+
+* ``auth_google.client_id``: client ID from Google developer console
+
+* ``auth_google.client_secret``: client secret from Google developer console
+
+Optional:
+
+* ``auth_google.allow_sign_up``: set to true to allow users to sign up via Google authentication. If enabled, each new user is created with "viewer" permission and added to their own newly-created organizations
+
+* ``user_auto_assign_org_role``: specify different permissions (Editor, Admin)
+
+* ``user_auto_assign_org``: set to true to add all new users to the main organization
+
+.. image:: /images/products/grafana/aiven_add_google_oauth.png

```

Comment:
Top tip: EVERY image needs an "alt" tag, otherwise the validation for the rest of the file is utter nonsense. We haven't worked out how to syntax check for alt tags, so you don't get a sensible error, sorry!

Reply:
(just a note that the images do need alt tags, please add these)